samwise.js

```diff
@@ -4,6 +4,7 @@
  * Copyright 2016 Henrik Ingo, henrik.ingo@avoinelama.fi
```

Comment:
SCRIPT-This function might benefit from using async/await instead of callbacks for better readability.